src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryCachingConfiguration.java

```diff
@@ -29,16 +29,19 @@ public final class LibraryCachingConfiguration extends AbstractDescribableImpl<L
     
     private int refreshTimeMinutes;
     private String excludedVersionsStr;
+    private String includedVersionsStr;
 
     private static final String VERSIONS_SEPARATOR = " ";
     public static final String GLOBAL_LIBRARIES_DIR = "global-libraries-cache";
     public static final String LAST_READ_FILE = "last_read";
 
-    @DataBoundConstructor public LibraryCachingConfiguration(int refreshTimeMinutes, String excludedVersionsStr) {
+    @DataBoundConstructor public LibraryCachingConfiguration(int refreshTimeMinutes, String excludedVersionsStr, String includedVersionsStr) {
```

Comment:
To keep backwards binary compatibility the old constructor signature should be kept and `@Deprecated` and call the new constructor with default values.

Reply:
Maybe even a better approach would be to instead just add a `@DataboundSetter` for the new field, so that it signals that it is optional and doesn't need to be provided unless needed.